name.py

```diff
@@ -0,0 +1,49 @@
+"""Extract and validate a given name and a family name."""
+import re
+from parse import parse
+
+def is_valid(name):
+    """Take in a string containing a name. Return True if it is valid,
+    Otherwise, return False.
+    """
+    if re.search("[a-zA-Z]+", name):
```

Comment:
one liner. you might want to refactor this pattern out, it's duplicated code. one shared function among all these validators of 'is_single_word' or something like that